src/git/models/reference.ts

```diff
@@ -101,15 +101,18 @@ export namespace GitRevision {
 
 export interface GitBranchReference {
 	readonly refType: 'branch';
+	id?: string;
 	name: string;
 	ref: string;
 	readonly remote: boolean;
 	readonly upstream?: { name: string; missing: boolean };
 	repoPath: string;
+	avatarUrl?: string;
```

Comment:
Same response as: https://github.com/gitkraken/GitKrakenComponents/pull/65#issuecomment-1268031596

We need `avatarUrl` on the VSCode DOM context to be able to set the `hiddenRefsById` when using Gitlens context menu to hide a branch.

Reply:
Then I think we should create a "graph only" wrapper for the `GitBranchReference` rather than adding it to the general model. Because `GitBranch` itself doesn't have an `avatarUrl` (and I'm not convinced that it should at this point)